syft/serde/serde.py

```diff
@@ -7,17 +7,13 @@
 
 from typing import Callable
 
-from syft.workers.abstract import AbstractWorker
-
-from syft.serde import msgpack
-
 ## SECTION:  High Level Public Functions (these are the ones you use)
 def serialize(
     obj: object,
-    worker: AbstractWorker = None,
+    worker=None,
     simplified: bool = False,
     force_full_simplification: bool = False,
-    strategy: Callable[[object, AbstractWorker], bin] = msgpack.serialize,
+    strategy: Callable[[object], bin] = None,
```

Comment:
It looks like this changes the call signature of the serialization strategies, but I'm not sure I saw a change to the strategies themselves. Do these still match up?

Reply:
Yes, I wanted to detach the AbstractWorker import from the file to prevent the circular import, I'll think for a way to bring this back.

Reply:
Makes sense re: avoiding a circular import. I just want to make sure these type annotations are correct(-ish). Maybe it could be `Callable[[object, object]], bin` for now?